SPECS/aspell/aspell.spec

```diff
@@ -1,25 +1,27 @@
-Summary: Spell checker
-Name: aspell
-Version: 0.60.8
-Release: 5%{?dist}
+Summary:        Spell checker
+Name:           aspell
+Version:        0.60.8
+Release:        6%{?dist}
 # LGPLv2+ .. common/gettext.h
 # LGPLv2  .. modules/speller/default/phonet.hpp,
 #            modules/speller/default/phonet.cpp,
 #            modules/speller/default/affix.cpp
 # GPLv2+  .. ltmain.sh, misc/po-filter.c
 # BSD     .. myspell/munch.c
-License: LGPLv2+ and LGPLv2 and GPLv2+ and BSD
+License:        LGPLv2+ AND LGPLv2 AND GPLv2+ AND BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
-URL: http://aspell.net/
-Source: ftp://ftp.gnu.org/gnu/aspell/aspell-%{version}.tar.gz
-
-Patch0: aspell-0.60.7-fileconflict.patch
-Patch1: aspell-0.60.7-pspell_conf.patch
-Patch2: aspell-0.60.7-mp.patch
-
-BuildRequires: gcc-c++
-BuildRequires: chrpath, gettext, ncurses-devel, pkgconfig, perl-interpreter
+URL:            http://aspell.net/
+Source:         ftp://ftp.gnu.org/gnu/aspell/aspell-%{version}.tar.gz
+Patch0:         aspell-0.60.7-fileconflict.patch
+Patch1:         aspell-0.60.7-pspell_conf.patch
+Patch2:         aspell-0.60.7-mp.patch
+BuildRequires:  chrpath
+BuildRequires:  gcc-c++
+BuildRequires:  gettext
+BuildRequires:  ncurses-devel
+BuildRequires:  perl-interpreter
+BuildRequires:  pkg-config
 
```

Comment:
There is a high CVE against ASPELL, but there is a patch available:
https://github.com/gnuaspell/aspell/commit/0718b375425aad8e54e1150313b862e4c6fd324a

https://nvd.nist.gov/vuln/detail/CVE-2019-25051